recipes/google-cloud-recommendations-ai/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "google-cloud-recommendations-ai" %}
+{% set version = "0.2.0" %}
```

Comment:
I'm starting with this version (which is not the latest) because it is needed by apache-beam with gcp
https://github.com/apache/beam/blob/v2.32.0/sdks/python/setup.py#L202

Thanks @dopplershift for pointing out the very obvious `<=0.2.0` that I missed.